um_canonicalize/tags.yaml

```diff
@@ -211,12 +240,14 @@ categories:
   Number:
     - SG
     - PL
+    - PC
     - GRPL
     - DU
     - TRI
     - PAUC
     - GPAUC
     - INVN
+    - SG+PL
```

Comment:
This would be in opposition to what?

Reply:
Yeah, this occurs only in the two Norwegians but shouldn't exist. It's not expressing both simultaneously; it's just saying that the adjective form is compatible with either.

Reply:
Such "+" combinations mostly occur due to syncretism (@aryamccarthy did you split cells merged due to syncretism?)
But In Mordvinic languages we have SG;PL  and multiple cases in case_comp (compounding), which is still not clear how to address. @kylebgorman I sent you this before, maybe you could comment on this (open an issue to discuss it?)